azure-pipelines.yml

```diff
@@ -57,7 +55,14 @@ phases:
           _BuildConfig: Debug
         release:
           _BuildConfig: Release
-          _DotNetPublishToBlobFeed: false
+          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), contains(variables['Build.DefinitionName'], 'official')) }}:
+            _DotNetPublishToBlobFeed: true
+            _PublishArgs: /p:PublishToSymbolServer=true
```

Comment:
@tmat - this seems like an OK workaround to me for now. This makes it possible to publish to BAR and symbol servers.

Reply:
Ah, cool :)

Reply:
Perhaps just file an issue to remove it and link it from here, so that we know it's not meant to be permanent.

Reply:
All of this is moving to pipelines right? When we solve this problem we'll remove all of this